desktop/install/mac-install.md

```diff
@@ -106,6 +106,10 @@ The `install` command accepts the following flags:
   - It must be used together with the `--allowed-org=<org name>` flag. 
   - For example:
     `--allowed-org=<org name> --admin-settings='{"configurationFileVersion": 2, "enhancedContainerIsolation": {"value": true, "locked": false}}'`
+- `--proxy-http-mode=<mode>`: HTTP Proxy mode, `system` (default) or `manual`.
```

Comment:
nit: other options are described using a sentence, e.g.
```suggestion
- `--proxy-http-mode=<mode>`: sets the HTTP Proxy mode, `system` (default) or `manual`.
```

(not related to this PR, but the descriptions should probably all start with a lowercase, e.g. `performs` instead of `Performs` and `automatically` instead of `Automatically`)

Reply:
Indeed, consistency is nice :D

Reply:
updated the rest of the flags